app/meetings/detail/-components/meeting-detail-header/template.hbs

```diff
@@ -0,0 +1,57 @@
+<h1 data-test-meeting-name>{{this.meeting.name}}</h1>
+
+<div data-test-meeting-location-and-date local-class='location-and-date-text'>
+    {{this.locationAndDateText}}
+</div>
+
+{{#if this.meeting.logoUrl}}
+    <div>
+        <img data-test-meeting-logo src={{this.meeting.logoUrl}} alt={{t 'meetings.detail.meeting-detail-header.image_alt'}} class='img-responsive'>
+    </div>
+{{/if}}
+
+{{#if this.meeting.active}}
+    <div local-class='toggle-panel-button'>
+        <OsfButton data-analytics-name='Toggle meeting submission panel' data-test-meeting-toggle-panel-button @type='link' @onClick={{action this.togglePanel}}>
```

Comment:
Please break this into multiple lines. A rule I've been using is break into multiple line if there are >1 attributes.